frontend/src/App.js

```diff
@@ -38,6 +64,8 @@ export default function App() {
 						removeDoctor={removeDoctor}
 						selectedRowParams={selectedRowParams}
 						setSelectedRowParams={setSelectedRowParams}
+						fabPosition={fabPosition}
+						addIconStyle={addIconStyle}
```

Comment:
there is no need to pass the styles as props to all of these components. You could as well just export them, either from here or imho better in a seperate helper file and then import them wherever you need them.